src/Chapter1.hs

```diff
@@ -631,9 +641,14 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 = positiveSumLast2 . abs
+  where
+    positiveSumLast2 :: Int -> Int
+    positiveSumLast2 n = last + secondLast
+      where
+        last = n `mod` 10
+        secondLast = ((`mod` 10).(`div` 10)) n
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod (abs n) 10
div (abs n) 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod n 10
```

You can see how we could pattern match on the pair 🙂

Also, in Haskell operators are separated with spaces 👾  So you can add them for `.` as well 🙂

Reply:
Ack - adjusted to
```
...
(divNum, lastNum) = n `divMod` 10
secondLastNum = divNum `mod` 10
```